src/utils/cordova/setupRefresh.js

```diff
@@ -1,41 +1,36 @@
-import Vue from 'vue'
+import { reactive, watch } from 'vue'
 import datastore from '@/store'
 
-const status = new Vue({
-  data () {
-    return {
-      foreground: true,
-      online: null,
-    }
-  },
-  created () {
-    this.online = navigator.connection.type !== 'none'
-    document.addEventListener('deviceready', () => {
-      document.addEventListener('online', () => {
-        this.online = true
-      }, false)
-      document.addEventListener('offline', () => {
-        this.online = false
-      }, false)
+const state = reactive({
+  foreground: true,
+  online: null,
+})
 
-      document.addEventListener('resume', () => {
-        this.foreground = true
-      }, false)
+state.online = navigator.connection.type !== 'none'
+document.addEventListener('deviceready', () => {
+  document.addEventListener('online', () => {
+    state.online = true
+  }, false)
+  document.addEventListener('offline', () => {
+    state.online = false
+  }, false)
 
-      document.addEventListener('pause', () => {
-        this.foreground = false
-      }, false)
-    }, false)
-  },
-})
+  document.addEventListener('resume', () => {
+    state.foreground = true
+  }, false)
+
+  document.addEventListener('pause', () => {
+    state.foreground = false
+  }, false)
+}, false)
```

Comment:
no, this is really just blind upgrading. I think I added a comment in the pull request notes though..

Reply:
ok, probably can do cordova testing after merge... my phone screen broke and it might be my "android testing" phone now, can access the screen using https://github.com/Genymobile/scrcpy